imgedit/nls/imgedit.tr

```diff
@@ -142,7 +155,8 @@ MKEY.EDIT.SEQ.DEL=
 
 MKEY.TOOLS=
 MKEY.IMAGE.CLEAR=t
-MKEY.IMAGE.INVERT=e
+MKEY.IMAGE.INVERMKEY.FILE.IMPORT=m

```

Comment:
Line 158&159 looks like a Copy/Paste Error.

Reply:
I'll just fix it in FreeDOS edit to keep from mangling the codepage characters. :-)

Reply:
Thank you and sorry for the copy/paste mistake. While trying to understand how I managed to paste something there, I read your comment stating:
"So, if your language
; specific shortcut requires a key not present on a standard english keyboard,
; you'll need to let me know so I can map it to that character code"

Please note that I used "MKEY.FILE.EXIT=ç" in Turkish, for the first letter of "Çık", meaning exit (the program). There's also "MKEY.FILE.IMPORT=İ". Should I change the shortcuts or will you map them to the relevant character codes?

Reply:
You can change to whatever makes the most sense. Just don't use the same one twice, or the UI will only ever see the first occurrence when looking up the keystroke. Also, letter not in the word can be used. For example, the Item Could say "Quit Program", and the hot-key could be "x". The UI would display "Quit Program (Alt+x)". If the hot-key is "q" it would only show "**Q**uit Program" (with the Q highlighted).

For testing purposes, if you place a the translation file in the same directory as the imgedit.exe, the program will use that file instead of the version that gets glued to the executable. Same goes for the Turkish font. Put, that in the same dir as the exe. Then tell ImgEdit 1012N-TR.FNT to edit it. Change it, Save, Restart and see it using the updated font.

Reply:
Thanks for the explanation. I'll just leave it with your corrected keyboard shortcuts, seems safer :) Btw, PR #64 is whining for your attention :-P